src/SDK/MonitoringApplianceEmulator/Models/Chart/DateTimeDataPoint.cs

```diff
@@ -0,0 +1,37 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="DateTimeDataPoint.cs" company="Microsoft Corporation">
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// </copyright>
+//-----------------------------------------------------------------------
+
+namespace Microsoft.Azure.Monitoring.SmartDetectors.MonitoringApplianceEmulator.Models.Chart
```

Comment:
do we really need a separate namespace just for this class?

Reply:
ok ok 

---
In reply to: [231015667](https://github.com/Azure/SmartDetectorsMonitoringAppliance/pull/17#discussion_r231015667) [](ancestors = 231015667)